apps/cli/src/app.ts

```diff
@@ -47,15 +50,15 @@ export function createApp(config?: AppConfig): App {
 
   const env = new Env(process.env as Record<string, string>);
 
-  // Generate a unique log file
+  // Chat Log File
   const date = new Date();
   const logFile = `chat_${date.getFullYear()}-${
     date.getMonth() + 1
   }-${date.getDate()}_${date.getHours()}-${date.getMinutes()}-${date.getSeconds()}.md`;
   const logWorkspace = new FileSystemWorkspace(path.join(rootDir, "chats"));
   const fileLogger = new FileLogger(logWorkspace.toWorkspacePath(logFile));
 
-  // Create logger
+  // Logger
```

Comment:
Yah agreed, just wanted some way to separate the different sections because (right now) some are not too trivial. Will improve the agent's initialization in the future.